app/serializers/park_show_serializer.rb

```diff
@@ -1,5 +1,19 @@
 class ParkShowSerializer < ActiveModel::Serializer
-  attributes :id, :name, :state, :description, :photo
+  attributes :id, :name, :state, :description, :photo, :average
 
   has_many :reviews
+
+  def average
+    review_total = []
+    object.reviews.each do |review|
+      review_total << review.rating
+    end
+    if review_total.length > 0
+      review_average = review_total.sum.to_f / review_total.length
+      review_average.round(1)
+      "#{review_average.round(1)}/5 stars"
+    else
+      return "Not Yet Rated"
+    end
+  end
```

Comment:
`#average` is now written out twice in two serializers.  At this point I would rather abstract this out to the model level, so that all I would need to call is `object.average_review_rating` or something to that affect.